src/components/typography/typography.spec.tsx

```diff
@@ -606,16 +606,20 @@ describe("Typography", () => {
     );
 
     it.each(["clip", "ellipsis", "string", "initial", "inherit"])(
-      "applies text-overflow of %s",
+      "applies text-overflow of %s when text is truncated",
```

Comment:
**suggestion** (non-blocking): could it be worth testing this in a separate test, as we might want to test the attribute is still correctly applied before `truncate` is passed and after

Reply:
"white-space: nowrap" and "overflow" hidden" have to be set in order for the "text-overflow" property to have any effect and these can be set by the `truncate` prop, so they will always have to be used together. The properties could be set individually, but that is currently not possible as we do not have a prop to set the "overflow" property.

Reply:
Yeah I didn't think it was worth adding more complexity to the styles if they could be just used together if needed